packages/map-template/src/components/Map/MapboxMap/MapboxMap.jsx

```diff
@@ -45,28 +45,28 @@ function MapboxMap({ onMapView, onPositionControl }) {
     // Add Floor Selector to the Map when ready.
     useEffect(() => {
         if (mapsIndoorsInstance && mapView && !hasFloorSelector) {
-            const floorSelectorDiv = document.createElement('div');
-            new mapsindoors.FloorSelector(floorSelectorDiv, mapsIndoorsInstance);
+            const floorSelectorElement = document.createElement('mi-floor-selector');
+            floorSelectorElement.mapsindoors = mapsIndoorsInstance;
+            floorSelectorElement.primaryColor = '#FFA500';
```

Comment:
Is this the color we want to use in the Map Template?

Reply:
@ammapspeople ah that's a good catch, we need the color the user pass in - Where can I get that?

Reply:
Take a look at how `primaryColorState` is used in the `LocationDetails.jsx`.